tasks/wn.yaml

```diff
@@ -1,25 +1,4 @@
 ---
-    # We have to remove the entries in /etc/hosts that correspond to the node name
-    - shell: cat /etc/hosts | sed "/{{slurm_server_name}}/d" > /tmp/newhosts  
-    # Now we add a new entry with the new IP address
-    - shell: echo "{{ slurm_server_ip }} {{ slurm_server_name }}" >> /tmp/newhosts 
-    # Let's set the new /etc/hosts file
-    - shell: cp /tmp/newhosts /etc/hosts && rm /tmp/newhosts 
-
-    - service: name=firewalld state=stopped
-      when: ansible_os_family == "RedHat" and ansible_distribution_major_version|int >= 7
-      ignore_errors: yes
-
```

Comment:
yep I should put back the firewalld stopped task. However I don't really understand what the first lines are doing to the hosts file. Perhaps we don't need it anymore with https://github.com/indigo-dc/tosca-types/pull/138 ? I had no trouble without them.